attesters/tsm/tsm.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package tsm
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+
+	"github.com/fxamacker/cbor/v2"
+	"github.com/google/go-configfs-tsm/configfs/linuxtsm"
+	"github.com/google/go-configfs-tsm/report"
+	"github.com/veraison/cmw"
+	"github.com/veraison/ratsd/api"
+	"github.com/veraison/ratsd/proto/compositor"
+)
+
+const (
+	ApplicationvndVeraisonConfigfsTsmCbor = "application/vnd.veraison.configfs-tsm+cbor"
+)
+
+var (
+	sid = &compositor.SubAttesterID{
+		Name:    "tsm-report",
+		Version: "1.0.0",
+	}
+
+	supportedFormats = []*compositor.Format{
+		&compositor.Format{
+			ContentType: string(api.ApplicationvndVeraisonConfigfsTsmJson),
```

Comment:
why does this need the wrapping and the similar assignment at L34 does not?

Reply:
`api.ApplicationvndVeraisonConfigfsTsmJson` has a type `ApplicationvndVeraisonConfigfsTsmJson`, which is incompatible with `Format.ContentType` defined in `compositor.proto`

Reply:
Thanks for the clarification.

My next question is: why is `ApplicationvndVeraisonConfigfsTsmJson` generated by the API? It shouldn’t, right?  I mean, it’s the responsibility of the plugin to declare its media types.

Reply:
This is because in `docs/api/ratsd.yaml`, there is an enumeration for type `application/vnd.veraison.configfs-tsm+json`
```yaml
CMW:
      type: object
      required:
        - typ
        - val
      properties:
        typ:
          type: string
          enum:
            - application/vnd.veraison.configfs-tsm+json
```
I'd suggest we get rid of the enum in OpenAPI yaml and define the content types in each of the attester.

Reply:
I've defined additional type `ApplicationvndVeraisonConfigfsTsmJson` in TSM attester, thus `api.ApplicationvndVeraisonConfigfsTsmJson` is no longer used and can be safely removed from `ratsd.yaml`